### PR TITLE
Intel Parallel Studio 2019.3

### PR DESCRIPTION
**Pull request by gyengen**

### CODE REVIEW

File: sharc/software/modulefiles/libs/intel-mkl/2019.3/binary [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+prepend-path LD_LIBRARY_PATH $intelpsroot/compilers_and_libraries_2019.3.199/linux/compiler/lib/intel64_lin;
+prepend-path LD_LIBRARY_PATH $intelpsroot/compilers_and_libraries_2019.3.199/linux/tbb/lib/intel64_lin/gcc4.7;
+setenv PKG_CONFIG_PATH $intelpsroot/compilers_and_libraries_2019.3.199/linux/mkl/bin/pkgconfig;
+prepend-path LIBRARY_PATH $intelpsroot/compilers_and_libraries_2019.3.199/linux/mkl/lib/intel64_lin;
+prepend-path LIBRARY_PATH $intelpsroot/compilers_and_libraries_2019.3.199/linux/compiler/lib/intel64_lin;
+prepend-path LIBRARY_PATH $intelpsroot/compilers_and_libraries_2019.3.199/linux/tbb/lib/intel64_lin/gcc4.7;
+setenv MKLROOT $intelpsroot/compilers_and_libraries_2019.3.199/linux/mkl;
+setenv NLSPATH $intelpsroot/compilers_and_libraries_2019.3.199/linux/mkl/lib/intel64_lin/locale/%l_%t/%N;
+
+setenv MKLROOT $intelpsroot/compilers_and_libraries_2017.0.098/linux/mkl
```
Comment (willfurnass) [inline sharc/software/modulefiles/libs/intel-mkl/2019.3/binary]: Wrong path

**Comment by willfurnass** [comment]
Looks good.  A couple of points:
 - the Parallel Studio docs page says DAAL is only available in PS 2017 when it's available in PS >=2017
 - the MKL modulefile sets `MKLROOT` to the wrong path

File: sharc/software/modulefiles/libs/intel-mkl/2019.3/binary (hunk 1 above)
Comment (gyengen) [reply sharc/software/modulefiles/libs/intel-mkl/2019.3/binary]: nice catch, thanks!

Comment (gyengen) [reply sharc/software/modulefiles/libs/intel-mkl/2019.3/binary]: we don't need that line

**Comment by willfurnass** [comment]
LGTM

**Pull request merged**